Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3735,6 +3748,11 @@ GstElement* MediaPlayerPrivateGStreamer::createVideoSinkGL()
         return nullptr;
     }
 
+#if PLATFORM(QCOM_DB)
+    sink = makeGStreamerElement("db410csink", "optimized vsink");
```

Comment:
Could this be done at runtime with an env var like `WEBKIT_GST_CUSTOM_VIDEO_SINK=mycooloptimizedsink`?

Reply:
Why would you want that? I guess it's their preference to have it this way.

Reply:
Reduced technical debt for us.